pkg/wasm/cache.go

```diff
@@ -375,17 +396,19 @@ func (c *LocalFileCache) purge() {
 		case <-ticker.C:
 			c.mux.Lock()
 			for k, m := range c.modules {
-				if m.expired(c.wasmModuleExpiry) {
-					// The module has not be touched for expiry duration, delete it from the map as well as the local dir.
-					if err := os.Remove(m.modulePath); err != nil {
-						wasmLog.Errorf("failed to purge Wasm module %v: %v", m.modulePath, err)
-					} else {
-						for downloadURL := range m.referencingURLs {
-							delete(c.checksums, downloadURL)
-						}
-						delete(c.modules, k)
-						wasmLog.Debugf("successfully removed stale Wasm module %v", m.modulePath)
+				if !m.expired(c.wasmModuleExpiry) {
+					continue
+				}
+				// The module has not be touched for expiry duration, delete it from the map as well as the local dir.
+				if err := os.Remove(m.modulePath); err != nil {
+					wasmLog.Errorf("failed to purge Wasm module %v: %v", m.modulePath, err)
+				} else {
+					for downloadURL := range m.referencingURLs {
+						delete(c.checksums, downloadURL)
 					}
+					delete(c.modules, k)
+					log.Infof("modules :%v", c.modules)
```

Comment:
is this extra log line helpful?  should it be `wasmLog.Infof`? or `wasmLog.Debugf`?

Reply:
Oh.. my bad. will remove it. Thanks!